spec/models/import/row_error_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe Import::RowError do
+  subject {
+    described_class.new(
+      row_number: 10,
+      column: "Column Header",
+      value: "The value",
+      message: "The message"
+    )
+  }
+
+  describe "#row_number" do
+    it "returns the supplied row_number" do
+      expect(subject.row_number).to eql 10
+    end
+  end
+
+  describe "#column" do
+    it "returns the supplied column header" do
+      expect(subject.column).to eql "Column Header"
+    end
+  end
+
+  describe "#value" do
+    it "returns the supplier value" do
+      expect(subject.value).to eql "The value"
+    end
+  end
+
+  describe "#csv_row" do
```

Comment:
I'd prefer if this method was named `csv_row_number`, to avoid confusion that it might refer to the CSV row being imported.

Reply:
Oh, this was named to be compatible with the existing importer and views - I've had to add it back in later! :(